File: Selection_sort/selection_sort.py
from typing import List

def selectionSort(arr: List[int]) -> None: 
    # Write your code here
    for i in range(len(arr)-1):
        mini=i
        for j in range(i+1,len(arr)):
            if(arr[mini]>arr[j]):
                mini=j
        temp=arr[i]
        arr[i]=arr[mini]
        arr[mini]=temp
    pass

File: Sorting_algorithms/selection_sort.java
public class Solution {
    public static void selectionSort(int[] arr) {
        //Your code goes here
        for (int i=0;i<arr.length-1;i++){
            int mini=i;
            for (int j=i+1;j<arr.length;j++){
                if(arr[mini]>arr[j]) {
                    mini=j;
                }
            }
            int temp=arr[i];
            arr[i]=arr[mini];
            arr[mini]=temp;
        }
    }
}